test/signs/realtime_test.exs

```diff
@@ -244,33 +228,33 @@ defmodule Signs.RealtimeTest do
 
     test "when sign is at a station closed by shuttles and there are no predictions, it says so" do
       expect(Engine.Alerts.Mock, :max_stop_status, fn _, _ -> :shuttles_closed_station end)
-      expect_messages({"No train service", "Use shuttle bus"})
```

Comment:
Does there remain any way to test the "route-less" logic now? Is it dead code?

Reply:
Almost, but not quite. The Ashmont mezzanine can still trigger this case. I added another case to exercise that.

Reply:
Nice, even better test coverage then!